packages/core/strapi/bin/strapi.js

```diff
@@ -314,6 +316,42 @@ program
       }
     }
   })
+  .hook('preAction', async (thisCommand) => {
+    const opts = thisCommand.opts();
+    const encrypt = path.extname(String(opts.file)) === '.enc';
+    const compress = ['.gz', '.gz.enc'].includes(path.extname(String(opts.file)));
```

Comment:
For the record, I don't like this and will probably improve it after the beta. Probably by writing something that tries opening the file in various ways and seeing which one succeeds, so that we don't have to rely on the extension at all. But that will take some time.